lib/v2/storyfm/episodes.js

```diff
@@ -0,0 +1,65 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+module.exports = async (ctx) => {
+    const rootUrl = 'https://storyfm.cn';
+    const currentUrl = `${rootUrl}/episodes-list`;
```

Comment:
```suggestion
    const currentUrl = `${rootUrl}/episodes-list/`;
```
https://github.com/DIYgod/RSSHub/actions/runs/4949534372/jobs/8851823208#step:12:17